reference/twig_reference.rst

```diff
@@ -371,18 +372,22 @@ absolute_url
 ``path``
     **type**: ``string``
 
-Returns the absolute URL for the given absolute path. This is useful to convert
-an existing path:
+Returns the absolute URL from the passed relative path. Assume the following
+code was used to render ``http://example.com/sub/list.html``:
```

Comment:
This is confusing me a little bit. Are we saying basically "Assume your Symfony installation is at http://example.com/sub`? What I mean is: are we trying to show that if you don't include `/`, then you should pass paths relative to your document root (e.g. the `web` dir) and that `absolute_url` will figure out if your installation is under a sub-directory and correct? If so, this is exactly how `asset()` works now - I'm just verifying that I'm not missing something. I think the fact that it corrects the subdirectory is not that important: it's just something that it should do. What's important is that we recommend (first) that you don't use the `/`, and that the path you should use is relative to your document root (e.g. `web/`).